src/components/MessagesList/MessagesList.vue

```diff
@@ -241,6 +241,10 @@ export default {
 		currentDay() {
 			return moment().startOf('day').unix()
 		},
+
+		isScrollable() {
```

Comment:
For me value is calculated only once for a long chat. When I switch to the short chat, issue is still persisted

Though approach is correct, we need to trigger it differently. `v-element-resize` is more reliable, but still quirky

Reply:
We can add `this.token` to trigger recalculation on conv switch or add `this.messagesList.length`  to also make sure that the issue won't occur when deleting chat messages.

Reply:
Still happens from time to time =/
![image](https://github.com/nextcloud/spreed/assets/93392545/69664797-f822-4c1a-9e72-e28ea5f864e5)

Reply:
Even with isScrollable faulty ?

Reply:
I'm thinking it needs nextTick to update the chat and it's height anyway